src/main/java/seedu/guest/model/guest/IsRoomClean.java

```diff
@@ -10,26 +10,51 @@
 public class IsRoomClean {
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Is Room Clean should only contain the strings \"yes\" or \"no\"";
-    public static final String VALIDATION_REGEX = "^yes$|^no$";
+            "IsRoomClean field should only contain any case-insensitive versions of the strings"
+                    + "\"yes\", \"no\", \"y\" or \"n\".";
+    public static final String VALIDATION_REGEX = "^yes$|^no$|^y$|^n$";
+    public static final String DEFAULT_IS_ROOM_CLEAN = "yes";
     public final String value;
 
     /**
      * Constructs a {@code IsRoomClean}.
      *
-     * @param isClean A valid is room clean.
+     * @param isRoomClean A valid isRoomClean.
      */
-    public IsRoomClean(String isClean) {
-        requireNonNull(isClean);
-        checkArgument(isValidIsRoomClean(isClean), MESSAGE_CONSTRAINTS);
-        value = isClean;
+    public IsRoomClean(String isRoomClean) {
+        requireNonNull(isRoomClean);
+        String isRoomCleanToLower = isRoomClean.toLowerCase();
```

Comment:
Is this unnecessary as `isValidIsRoomClean` already does it?

Reply:
Yes, you are right. However, isValidRoomClean does not return the lowercased string. So I added the toLowerCase method to operate on the string in the standardiseIsRoomClean instead.